src/generate_chapters.py

```diff
@@ -0,0 +1,63 @@
+import os
+import re
+import mistune
+import yaml
+
+renderer = mistune.Renderer()
+markdown = mistune.Markdown(renderer=renderer)
+
+# This could perhaps be loaded from an external file.
```

Comment:
I agree, it makes sense. I think I've implemented it as per your suggestions, but the `<lang>/<year>/chapter.html` files look almost identical other than the extends. Would you check to see if this is what you had in mind?

Reply:
For now the templates look similar because we haven't implemented any of the more complicated rendering. I'd imagine we would want to do something like show a special byline for the authors.

The advantage to having one template that is generated and one that isn't is that we can update the base chapter page look and feel without having to regenerate all of its child templates.